packages/tools/eslint-config/package.json

```diff
@@ -27,6 +27,10 @@
   "devDependencies": {
     "@rushstack/eslint-config": "~3.3.0",
     "eslint": "^8.15.0",
+    "eslint-config-prettier": "~8.8.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jsx-a11y": "~6.7.1",
+    "eslint-plugin-prettier": "~4.2.1",
```

Comment:
Will adding them to `devDependencies` only work when this config is used in other packages? Do those packages need to add those dependencies themselves, or can we move this to `dependencies` and will it work as well? Or will it _not_ anyway, whether we put them as regular- _or_ devdependencies?

Reply:
no, it will work both in dependencies and devDependencies
the reason I have put them in devDependencies is just that they are dev packages

Reply:
Yes but for the eslint-config they're actual dependencies, as you install `@kadena/eslint-config` for the configuration specifically